test/unit/app/common/lib/suggestionTest.js

```diff
@@ -58,6 +58,11 @@ describe('suggestion unit tests', function () {
     it('normalizes location', function () {
       assert.ok(suggestion.normalizeLocation('https://www.site.com') === 'site.com', 'www. prefix removed')
       assert.ok(suggestion.normalizeLocation('http://site.com') === 'site.com', 'location not modified')
+      assert.ok(suggestion.normalizeLocation('http://wowww.com') === 'wowww.com', 'location not modified')
```

Comment:
these tests should not all say 'location not modified'

please also add one for `www.wowww.com`, thanks!

Reply:
oh, good catch! thanks. i'll make the changes now.